libstatistics_collector/package.xml

```diff
@@ -12,11 +12,17 @@
   <depend>rcl</depend>
   <depend>rcpputils</depend>
 
+  <build_depend>rosidl_default_generators</build_depend>
+  <build_depend>std_msgs</build_depend>
+
+  <exec_depend>rosidl_default_runtime</exec_depend>
+  <exec_depend>std_msgs</exec_depend>
+
   <test_depend>ament_cmake_gtest</test_depend>
   <test_depend>ament_lint_auto</test_depend>
   <test_depend>ament_lint_common</test_depend>
-  <test_depend>sensor_msgs</test_depend>
-  <test_depend>std_msgs</test_depend>
+
+  <member_of_group>rosidl_interface_packages</member_of_group>
```

Comment:
needed?

Reply:
Yes. Fails to compile otherwise (see [rosidl_generate_interfaces.cmake](https://github.com/ros2/rosidl/blob/66f9a0e92962778edddc2ce6522ec259d7a76331/rosidl_cmake/cmake/rosidl_generate_interfaces.cmake#L229-L232)).